examples/build-nginx.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
 
+if [ "$EUID" -ne 0 ]
+    then
+    echo "This script must be run as root."
```

Comment:
perhaps "This script uses functionality (run) which requires root privileges" or something?